src/scripts/index.js

```diff
@@ -1,6 +1,12 @@
+// const { DateTime } = require('luxon');
 const addButton = document.querySelector('.add-btn');
 const bookList = document.querySelector('.books-ul');
 const storage = window.localStorage;
+// const elem = document.querySelector('.main');
+const navLinks = document.querySelectorAll('.nav-ul a');
```

Comment:
- [ ] It would be better to remove old pieces of code to follow [the JavaScript best practices (#7)](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md). ♻️